### PR TITLE
SignatureRequestHandler usage and other

### PR DESCRIPTION
**Pull request by aridder**
This PR is trying to illustrate how to use signatureRequestHandler.

Also did an effort to give a clearer message to user when signing is needed in Symfoni ID. Look image below.

CleanShot 2021-11-25 at 13 46 23

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "graphql-hooks": "^5.4.1",
     "grommet": "^2.18.0",
     "grommet-icons": "^4.6.2",
+    "neverthrow": "^4.3.1",
```
Comment (Arxcis) [inline package.json:29]: 👀 New dep

File: src/App.tsx [hunk 2]
```diff
@@ -1,100 +1,92 @@
...
+                                        <Route path="/capTable/:address" component={CapTablePage} />
+                                        <Route path="/register" component={CapTableRegistryPage} />
+                                        <Route path="/que" component={CapTableQuePage} />
+                                        <Route path="/me" component={MePage} />
+                                        <Route path="/logout" component={LogoutPage} />
+                                    </Switch>
+                                </Main>
+                                {/** Permission request switch */}
+                                {/* <VerifiablePresentationRequestSwitch /> */}
+                                <SignatureRequestModal />
```
Comment (Arxcis) [inline src/App.tsx]: Commented out  `{/* <VerifiablePresentationRequestSwitch /> */}` and added `<SignatureRequestModal />`

File: src/App.tsx [hunk 3]
```diff
@@ -1,100 +1,92 @@
...
+                </Grommet>
+            </ClientContext.Provider>
+        </BrowserRouter>
+    );
 }
 
-function VerifiablePresentationRequestSwitch() {
-  const location = useLocation();
-  const history = useHistory();
+function VerifiablePresentationRequestSwitch() {
```
Comment (Arxcis) [inline src/App.tsx]: Remember to delete this.

File: src/context/useSymfoni.ts [hunk 4]
```diff
@@ -111,6 +111,54 @@ export const useSymfoni = () => {
         [lazySigner, selectedSigner]
     );
 
+    const signAccessVP = async () => {
```
Comment (Arxcis) [inline src/context/useSymfoni.ts]: I think we should make a new hook for the sign-functions called `useSignatureRequests()`.

- `useSymfoni()` is already complicated enough.
- There is a clear separation of responsibility to be made. Maintaining the connection with Symfoni could be the responsibility of `useSymfoni()`. 
- Making requests over said connection could be the responsibility of a the new proposed `useSignatureRequests()`-hook.

File: src/utils/SignerRequestHandler.ts [hunk 5]
```diff
@@ -56,7 +58,15 @@ export class SignatureRequestHandler {
             this.events.on("rejected", (event) => this.handleRejected(() => reject(event)));
         });
     }
-
+    async resultsNT<T>() {
```
Comment (Arxcis) [inline src/utils/SignerRequestHandler.ts:61]: This looks like dead code to me. Is it? Maybe there is some magic happening with the new package "neverthrow"

**Review by Arxcis — CHANGES_REQUESTED** [review]
Well played @aridder 👍 I would love to go through this together with you in person, when you have time for it 😉

**Review by Arxcis — CHANGES_REQUESTED** [review]
**FIXED!**

File: src/context/useSymfoni.ts (hunk 4 above)
Comment (aridder) [reply src/context/useSymfoni.ts]: Enig. Kan du hjelpe med dette @Arxcis ? 😇

File: src/App.tsx (hunk 3 above)
Comment (Arxcis) [reply src/App.tsx]: Remembered to delete it in 4ef6c75. Please review @aridder

File: src/context/useSymfoni.ts (hunk 4 above)
Comment (Arxcis) [reply src/context/useSymfoni.ts]: Fix klar i 3eb42e4. Please review @aridder 🙏

Comment (aridder) [reply src/context/useSymfoni.ts]: Supert 😄

**Review by Arxcis — CHANGES_REQUESTED** [review]
**FIXED**!

**Review by Arxcis — CHANGES_REQUESTED** [review]
**FIXED!**

**Pull request merged**